src/main/java/bio/terra/service/filedata/flight/ingest/ValidateIngestFileDirectoryStep.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.service.filedata.flight.ingest;
+
+import bio.terra.model.FileLoadModel;
+import bio.terra.service.dataset.Dataset;
+import bio.terra.service.filedata.exception.FileAlreadyExistsException;
+import bio.terra.service.filedata.exception.FileSystemAbortTransactionException;
+import bio.terra.service.filedata.flight.FileMapKeys;
+import bio.terra.service.filedata.google.firestore.FireStoreDao;
+import bio.terra.service.filedata.google.firestore.FireStoreDirectoryEntry;
+import bio.terra.service.job.JobMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ValidateIngestFileDirectoryStep implements Step {
+    private static final Logger logger = LoggerFactory.getLogger(ValidateIngestFileDirectoryStep.class);
+
+    private final FireStoreDao fileDao;
+    private final Dataset dataset;
+    public static final String createEntryAction = "createEntry";
+    public static final String checkEntryAction = "checkEntry";
```

Comment:
In Java, static final fields are usually named all caps
```suggestion
    public static final String CREATE_ENTRY_ACTION = "createEntry";
    public static final String CHECK_ENTRY_ACTION = "checkEntry";
```

Also normally I would define static fields above non-static fields in a class, but I'm not sure if the jade code follows that convention or not.